alda-mode.el

```diff
@@ -277,6 +277,29 @@ Because alda runs in the background, the only way to do this is with alda restar
   ;; Set alda highlighting
   (setq font-lock-defaults '(alda-highlights)))
 
+(defun alda-play-block ()
+  (interactive)
+  (let* ((p (point)))
+	(mark-paragraph)
+	(alda-play-region (region-beginning) (region-end))
+	(goto-char p)))
+
+(defun alda-play-line ()
+  (interactive)
+  (alda-play-region (line-beginning-position) (line-end-position)))
+
+(defun alda-play-buffer ()
+  (interactive)
```

Comment:
Can you replace this function with:

```
(defun alda-play-buffer ()
  (interactive)
  (alda-play-text (buffer-string)))
```
I'd rather not select the entire buffer to read it in.
Let me know if that breaks your use case.